src/components/projects_home/createExpense.tsx

```diff
@@ -1,9 +1,54 @@
-import React from "react";
```

Comment:
The component responsible for the creating of expenses, at the moment it doesn't send the data to the API, that will be addressed in the next push.